### PR TITLE
Update alert-types.mdx

### PR DESCRIPTION
**Pull request by lizokm**
Adding descriptions to metric alert types.

### CODE REVIEW

File: src/docs/product/alerts/alert-types.mdx [hunk 1]
```diff
@@ -58,26 +58,69 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
 
-- Crash Free Session Rate
-- Crash Free User Rate
+#### Crash Free User Rate
+
+Alerts when the overall user experience dips below the threshold you set.
+
+Crash Free Users represents the percentage of individual users who haven’t experienced a crash.
 
 ### Performance
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:87]:
```suggestion
### Performance

Application performance alerts can help you pinpoint and identify specific problems that could be causing a suboptimal experience for your users.
```

File: src/docs/product/alerts/alert-types.mdx [hunk 2]
```diff
@@ -58,26 +58,69 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+
+#### Failure Rate
+
+Alerts on failure rate (the percentage of unsuccessful transactions).
+
+Sentry treats transactions with a status other than “ok,” “canceled,” and “unknown” as failures.
+
+#### Largest Contentful Display
+
+Largest Contentful Paint (LCP) measures loading performance. It marks the point when the largest image or text block is visible within the viewport. A fast LCP helps reassure the user that the page is useful. We recommend an LCP of less than 2.5 seconds.
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:111]:
```suggestion
Alerts when the Largest Contentful Paint (LCP), which measures loading performance, is loading slower than expected. 

LCP marks the point when the largest image or text block is visible within the viewport. A fast LCP helps reassure the user that the page is useful. We recommend an LCP of less than 2.5 seconds.
```

File: src/docs/product/alerts/alert-types.mdx [hunk 3]
```diff
@@ -58,26 +58,69 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+
+#### First Input Delay
+
+First Input Delay (FID) measures interactivity as the response time when the user tries to interact with the viewport. A low FID helps ensure that a page is useful, and we recommend a FID of less than 100 milliseconds.
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:115]:
```suggestion
Alerts when First Input Delay (FID), which measures the response time when a user tries to interact with the viewport, is longer than expected. 

A low FID helps ensure that a page is useful. We recommend a FID of less than 100 milliseconds.
```

File: src/docs/product/alerts/alert-types.mdx [hunk 4]
```diff
@@ -58,26 +58,69 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+
+#### Cumulative Layout Shift
+
+Cumulative Layout Shift (CLS) measures visual stability by quantifying unexpected layout shifts that occur during the entire lifespan of the page. A CLS of less than 0.1 is a good user experience, while anything greater than 0.25 is poor.
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:119]:
```suggestion
Alerts when cumulative Layout Shift (CLS), which measures visual stability by quantifying unexpected layout shifts that occur during the entire lifespan of the page, increases. 

A CLS of less than 0.1 translates to a good user experience, while anything greater than 0.25 is poor.
```

File: src/docs/product/alerts/alert-types.mdx [hunk 5]
```diff
@@ -58,26 +58,69 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+
+#### Custom Metric
+
+Alert on metrics which are not listed above, such as first paint (FP), first contentful paint (FCP) and time to first byte (TTFB).
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:123]:
```suggestion
Custom alerts that can be set up to be triggered by metrics which are not listed above, such as first paint (FP), first contentful paint (FCP) and time to first byte (TTFB).
```

File: src/docs/product/alerts/alert-types.mdx [hunk 6]
```diff
@@ -58,26 +58,77 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
-- Custom Metric
+Application performance alerts can help you pinpoint and identify specific problems that could be causing a suboptimal experience for your users.
+
+#### Throughput
+
+Alerts when throughput (the total number of transactions in a project) reaches a threshold set by you within a specified period of time.
+
+#### Transaction Duration
+
+This alert can be set to monitor how long it takes for transactions to complete. It can be customized with flexible aggregates like percentiles, averages, and min/max.
```
Comment (shanamatthews) [inline src/docs/product/alerts/alert-types.mdx:97]: Should we rephrase this to start with "Alerts on..." to match the others?

```suggestion
Alerts when transaction durations meet certain conditions. The conditions can be customized with flexible aggregates like percentiles, averages, and min/max.
```

**Review by shanamatthews — APPROVED** [review]

File: src/docs/product/alerts/alert-types.mdx [hunk 7]
```diff
@@ -58,26 +58,77 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
 
 ### Errors
 
-- Number of Errors
-- Users Experiencing Errors
+Error alerts are useful for monitoring the overall level of errors in your project, errors occurring in specific parts of your app, or errors affecting your users.
+
+#### Number of Errors
```
[the remark this answers is not in the document]
Comment (lizokm) [reply src/docs/product/alerts/alert-types.mdx:63]: Adding Level 4 headers is more of an accessibility consideration. It helps screen readers "read" in the right order and also translate the level of importance :)

File: src/docs/product/alerts/alert-types.mdx (hunk 6 above)
Comment (lizokm) [reply src/docs/product/alerts/alert-types.mdx:97]: Great point!

File: src/docs/product/alerts/alert-types.mdx (hunk 7 above)
[the remark this answers is not in the document]
Comment (shanamatthews) [reply src/docs/product/alerts/alert-types.mdx:63]: I feel that these headers need to be distinguished though. It's not immediately obvious that "Errors" is a section header for all the following specific types of error-related alerts:


Screenshot 2023-10-02 at 5 03 15 PM

[the remark this answers is not in the document]
Comment (shanamatthews) [reply src/docs/product/alerts/alert-types.mdx:63]: One idea is to bold the L3 headers?

Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:63]:
```suggestion
-  **Number of Errors**
```

File: src/docs/product/alerts/alert-types.mdx [hunk 8]
```diff
@@ -58,26 +58,77 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+
+Alerts when the number of errors in a project matching your filters crosses the threshold you set.
+
+#### Users Experiencing Errors
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:67]:
```suggestion
-  **Users Experiencing Errors**
```

File: src/docs/product/alerts/alert-types.mdx [hunk 9]
```diff
@@ -58,26 +58,77 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+
+Alerts when the number of users affected by errors in your project crosses the threshold you set.
 
 ### Sessions (Crash Rate Alerts)
 
-Crash rate alerts tell you when the crash free percentage for either [sessions or users](/product/releases/health/#active-sessionsusers) falls below a specific threshold. This could happen because of a spike in the number of session or user [crashes](/product/releases/health/#crash). Use crash rate alerts to monitor the health of your application per project, or in a specific release
+Crash rate alerts can give you a better picture of the health of your app on a per project basis or in a specific release. They trigger when the crash-free percentage for either [sessions or users](/product/releases/health/#active-sessionsusers) falls below a specific threshold. This could happen because of a spike in the number of session or user [crashes](/product/releases/health/#crash).
+
+#### Crash Free Session Rate
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:75]:
```suggestion
- **Crash Free Session Rate**
```

File: src/docs/product/alerts/alert-types.mdx [hunk 10]
```diff
@@ -58,26 +58,77 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+
+Alerts when the number of crashed sessions exceeds the threshold you set.
+
+A session begins when a user starts the application and ends when it’s closed or sent to the background. A crash is when a session ends due to an error.
+
+#### Crash Free User Rate
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:81]:
```suggestion
- **Crash Free User Rate**
```

File: src/docs/product/alerts/alert-types.mdx [hunk 11]
```diff
@@ -58,26 +58,77 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
-- Transaction Duration
-- Apdex
-- Failure Rate
-- Largest Contentful Display
-- First Input Delay
-- Cumulative Layout Shift
-- Custom Metric
+Application performance alerts can help you pinpoint and identify specific problems that could be causing a suboptimal experience for your users.
+
+#### Throughput
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:91]:
```suggestion
-  **Throughput**
```

File: src/docs/product/alerts/alert-types.mdx [hunk 12]
```diff
@@ -58,26 +58,77 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+
+Alerts when throughput (the total number of transactions in a project) reaches a threshold set by you within a specified period of time.
+
+#### Transaction Duration
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:95]:
```suggestion
- **Transaction Duration**
```

File: src/docs/product/alerts/alert-types.mdx [hunk 13]
```diff
@@ -58,26 +58,77 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+
+Alerts when transaction durations meet certain conditions. The conditions can be customized with flexible aggregates like percentiles, averages, and min/max.
+
+#### Apdex
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:99]:
```suggestion
-  **Apdex**
```

File: src/docs/product/alerts/alert-types.mdx [hunk 14]
```diff
@@ -58,26 +58,77 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+
+Alerts on the Apdex score (the ratio of satisfactory, tolerable, and frustrated requests in a specific transaction or endpoint).
+
+Apdex is a metric used to track and measure user satisfaction based on your application response times.
+
+#### Failure Rate
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:105]:
```suggestion
-  **Failure Rate**
```

File: src/docs/product/alerts/alert-types.mdx [hunk 15]
```diff
@@ -58,26 +58,77 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+
+Alerts on failure rate (the percentage of unsuccessful transactions).
+
+Sentry treats transactions with a status other than “ok,” “canceled,” and “unknown” as failures.
+
+#### Largest Contentful Display
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:111]:
```suggestion
-  **Largest Contentful Display**
```

File: src/docs/product/alerts/alert-types.mdx [hunk 16]
```diff
@@ -58,26 +58,77 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+
+Alerts when the Largest Contentful Paint (LCP), which measures loading performance, is loading slower than expected.
+
+LCP marks the point when the largest image or text block is visible within the viewport. A fast LCP helps reassure the user that the page is useful. We recommend an LCP of less than 2.5 seconds.
+
+#### First Input Delay
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:117]:
```suggestion
-  **First Input Delay**
```

File: src/docs/product/alerts/alert-types.mdx [hunk 17]
```diff
@@ -58,26 +58,77 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+
+Alerts when First Input Delay (FID), which measures the response time when a user tries to interact with the viewport, is longer than expected.
+
+A low FID helps ensure that a page is useful. We recommend a FID of less than 100 milliseconds.
+
+#### Cumulative Layout Shift
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:123]:
```suggestion
-  **Cumulative Layout Shift**
```

File: src/docs/product/alerts/alert-types.mdx [hunk 18]
```diff
@@ -58,26 +58,77 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+
+Alerts when cumulative Layout Shift (CLS), which measures visual stability by quantifying unexpected layout shifts that occur during the entire lifespan of the page, increases.
+
+A CLS of less than 0.1 translates to a good user experience, while anything greater than 0.25 is poor.
+
+#### Custom Metric
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:129]:
```suggestion
-  **Custom Metric**
```

File: src/docs/product/alerts/alert-types.mdx [hunk 19]
```diff
@@ -58,26 +58,77 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
 
-- Throughput
-- Transaction Duration
-- Apdex
-- Failure Rate
-- Largest Contentful Display
-- First Input Delay
-- Cumulative Layout Shift
-- Custom Metric
+Application performance alerts can help you pinpoint and identify specific problems that could be causing a suboptimal experience for your users.
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:89]:
```suggestion
Application performance alerts can help you pinpoint and identify specific problems that may be causing a suboptimal experience for your users.
```

File: src/docs/product/alerts/alert-types.mdx [hunk 20]
```diff
@@ -58,26 +58,77 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
-- Users Experiencing Errors
+Error alerts are useful for monitoring the overall level of errors in your project, errors occurring in specific parts of your app, or errors affecting your users.
+
+- **Number of Errors**
+
+Alerts when the number of errors in a project matching your filters crosses the threshold you set.
+
+- **Users Experiencing Errors**
+
+Alerts when the number of users affected by errors in your project crosses the threshold you set.
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:69]:
```suggestion
- **Number of Errors**
Alerts when the number of errors in a project matching your filters crosses the threshold you set.

- **Users Experiencing Errors**
Alerts when the number of users affected by errors in your project crosses the threshold you set.
```

File: src/docs/product/alerts/alert-types.mdx [hunk 21]
```diff
@@ -58,26 +58,77 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
 ### Sessions (Crash Rate Alerts)
 
-Crash rate alerts tell you when the crash free percentage for either [sessions or users](/product/releases/health/#active-sessionsusers) falls below a specific threshold. This could happen because of a spike in the number of session or user [crashes](/product/releases/health/#crash). Use crash rate alerts to monitor the health of your application per project, or in a specific release
+Crash rate alerts can give you a better picture of the health of your app on a per project basis or in a specific release. They trigger when the crash-free percentage for either [sessions or users](/product/releases/health/#active-sessionsusers) falls below a specific threshold. This could happen because of a spike in the number of session or user [crashes](/product/releases/health/#crash).
+
+- **Crash Free Session Rate**
+
+Alerts when the number of crashed sessions exceeds the threshold you set.
+
+A session begins when a user starts the application and ends when it’s closed or sent to the background. A crash is when a session ends due to an error.
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:79]:
```suggestion
- **Crash Free Session Rate**
Alerts when the number of crashed sessions exceeds the threshold you set. A session begins when a user starts the application and ends when it’s closed or sent to the background. A crash is when a session ends due to an error.
```

File: src/docs/product/alerts/alert-types.mdx [hunk 22]
```diff
@@ -58,26 +58,77 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+
+- **Crash Free User Rate**
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:81]:
```suggestion
- **Crash Free User Rate**
Alerts when the overall user experience dips below the threshold you set. Crash Free Users represents the percentage of individual users who haven’t experienced a crash.
```

File: src/docs/product/alerts/alert-types.mdx [hunk 23]
```diff
@@ -58,26 +58,77 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
 
-- Crash Free Session Rate
-- Crash Free User Rate
+Alerts when the overall user experience dips below the threshold you set.
+
+Crash Free Users represents the percentage of individual users who haven’t experienced a crash.
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:85]:
```suggestion

```

File: src/docs/product/alerts/alert-types.mdx [hunk 24]
```diff
@@ -58,26 +58,77 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+#### Transaction Duration
+
+Alerts when transaction durations meet certain conditions. The conditions can be customized with flexible aggregates like percentiles, averages, and min/max.
+
+#### Apdex
+
+Alerts on the Apdex score (the ratio of satisfactory, tolerable, and frustrated requests in a specific transaction or endpoint).
+
+Apdex is a metric used to track and measure user satisfaction based on your application response times.
+
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:104]:
```suggestion
- **Throughput**
Alerts when throughput (the total number of transactions in a project) reaches a threshold set by you within a specified period of time.

- **Transaction Duration**
Alerts when transaction durations meet certain conditions. 

**TIP:** The conditions can be customized with flexible aggregates like percentiles, averages, and min/max.

- **Apdex**
Alerts on the Apdex score (the ratio of satisfactory, tolerable, and frustrated requests in a specific transaction or endpoint). Apdex is a metric used to track and measure user satisfaction based on your application response times.

```

File: src/docs/product/alerts/alert-types.mdx [hunk 25]
```diff
@@ -58,26 +58,77 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+Alerts when the Largest Contentful Paint (LCP), which measures loading performance, is loading slower than expected.
+
+LCP marks the point when the largest image or text block is visible within the viewport. A fast LCP helps reassure the user that the page is useful. We recommend an LCP of less than 2.5 seconds.
+
+- **First Input Delay**
+
+Alerts when First Input Delay (FID), which measures the response time when a user tries to interact with the viewport, is longer than expected.
+
+A low FID helps ensure that a page is useful. We recommend a FID of less than 100 milliseconds.
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:121]:
```suggestion
- **Failure Rate**
Alerts on failure rate (the percentage of unsuccessful transactions). Sentry treats transactions with a status other than “ok,” “canceled,” and “unknown” as failures.

- **Largest Contentful Display**
Alerts when the Largest Contentful Paint (LCP), which measures loading performance, is loading slower than expected. LCP marks the point when the largest image or text block is visible within the viewport. 

**TIP:** A fast LCP helps reassure the user that the page is useful. We recommend an LCP of less than 2.5 seconds.

- **First Input Delay**
Alerts when First Input Delay (FID), which measures the response time when a user tries to interact with the viewport, is longer than expected. 

**TIP:** A low FID helps ensure that a page is useful. We recommend a FID of less than 100 milliseconds.
```

File: src/docs/product/alerts/alert-types.mdx [hunk 26]
```diff
@@ -58,26 +58,61 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+
+- **Users Experiencing Errors**
+  Alerts when the number of users affected by errors in your project crosses the threshold you set.
 
 ### Sessions (Crash Rate Alerts)
 
-Crash rate alerts tell you when the crash free percentage for either [sessions or users](/product/releases/health/#active-sessionsusers) falls below a specific threshold. This could happen because of a spike in the number of session or user [crashes](/product/releases/health/#crash). Use crash rate alerts to monitor the health of your application per project, or in a specific release
+Crash rate alerts can give you a better picture of the health of your app on a per project basis or in a specific release. They trigger when the crash-free percentage for either [sessions or users](/product/releases/health/#active-sessionsusers) falls below a specific threshold. This could happen because of a spike in the number of session or user [crashes](/product/releases/health/#crash).
+
+- **Crash Free Session Rate**
```
Comment (isabellaenriquez) [inline src/docs/product/alerts/alert-types.mdx:73]: Since these aren't headers of any sort, I assume we can't link to them right? In that case, are we thinking of just linking users to this page from the API documentation (as opposed to linking each metric alert type directly to its section on this page)?

**Review by isabellaenriquez — COMMENTED** [review]
just had a question regarding the headers! thanks so much!

File: src/docs/product/alerts/alert-types.mdx [hunk 27]
```diff
@@ -42,7 +42,7 @@ The **Alert Details** page also includes a list of issues that triggered the ale
 
 <Include name="feature-available-for-plan-team.mdx" />
 
-Metric alerts tell you when a [metric](/product/performance/metrics/) crosses a threshold, such as a spike in the number of errors in a project, or a change in a performance metric, like [latency](/product/performance/metrics/#latency), [Apdex](/product/performance/metrics/#apdex), [failure rate](/product/performance/metrics/#failure-rate), or [throughput](/product/performance/metrics/#throughput-total-tpm-tps).
+Metric alerts tell you when a [metric](/product/performance/metrics/) crosses a threshold set by you, such as a spike in the number of errors in a project, or a change in a performance metric, like [transaction duration](/product/performance/metrics/#latency), [Apdex](/product/performance/metrics/#apdex), [failure rate](/product/performance/metrics/#failure-rate), or [throughput](/product/performance/metrics/#throughput-total-tpm-tps).
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:45]:
```suggestion
Metric alerts tell you when a [metric](/product/performance/metrics/) crosses a threshold set by you, like a spike in the number of errors in a project, or a change in a performance metric, such as [transaction duration](/product/performance/metrics/#latency), [Apdex](/product/performance/metrics/#apdex), [failure rate](/product/performance/metrics/#failure-rate), or [throughput](/product/performance/metrics/#throughput-total-tpm-tps).
```

File: src/docs/product/alerts/alert-types.mdx [hunk 28]
```diff
@@ -58,26 +58,61 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+- **First Input Delay**
+  Alerts when First Input Delay (FID), which measures the response time when a user tries to interact with the viewport, is longer than expected.
+
+**TIP:** A low FID helps ensure that a page is useful. We recommend a FID of less than 100 milliseconds.
+
+#### Cumulative Layout Shift
+
+Alerts when cumulative Layout Shift (CLS), which measures visual stability by quantifying unexpected layout shifts that occur during the entire lifespan of the page, increases.
+
+A CLS of less than 0.1 translates to a good user experience, while anything greater than 0.25 is poor.
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:111]:
```suggestion
- **Cumulative Layout Shift:**
Alerts when cumulative Layout Shift (CLS), which measures visual stability by quantifying unexpected layout shifts that occur during the entire lifespan of the page, increases.

**TIP:** A CLS of less than 0.1 translates to a good user experience, while anything greater than 0.25 is poor.
```

File: src/docs/product/alerts/alert-types.mdx [hunk 29]
```diff
@@ -58,26 +58,61 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+
+#### Custom Metric
+
+Alert on metrics which are not listed above, such as first paint (FP), first contentful paint (FCP) and time to first byte (TTFB).
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:115]:
```suggestion
- **Custom Metric:**
Alerts on metrics which are not listed above, such as first paint (FP), first contentful paint (FCP) and time to first byte (TTFB).
```

File: src/docs/product/alerts/alert-types.mdx [hunk 30]
```diff
@@ -58,26 +58,61 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
 
-- Number of Errors
-- Users Experiencing Errors
+Error alerts are useful for monitoring the overall level of errors in your project, errors occurring in specific parts of your app, or errors affecting your users.
+
+- **Number of Errors**
+  Alerts when the number of errors in a project matching your filters crosses the threshold you set.
+
+- **Users Experiencing Errors**
+  Alerts when the number of users affected by errors in your project crosses the threshold you set.
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:67]:
```suggestion
- **Number of Errors:**
  Alerts when the number of errors in a project matching your filters crosses the threshold you set.

- **Users Experiencing Errors:**
  Alerts when the number of users affected by errors in your project crosses the threshold you set.
```

File: src/docs/product/alerts/alert-types.mdx (hunk 26 above)
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:73]:
```suggestion
- **Crash Free Session Rate:**
```

File: src/docs/product/alerts/alert-types.mdx [hunk 31]
```diff
@@ -58,26 +58,61 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
 
-Crash rate alerts tell you when the crash free percentage for either [sessions or users](/product/releases/health/#active-sessionsusers) falls below a specific threshold. This could happen because of a spike in the number of session or user [crashes](/product/releases/health/#crash). Use crash rate alerts to monitor the health of your application per project, or in a specific release
+Crash rate alerts can give you a better picture of the health of your app on a per project basis or in a specific release. They trigger when the crash-free percentage for either [sessions or users](/product/releases/health/#active-sessionsusers) falls below a specific threshold. This could happen because of a spike in the number of session or user [crashes](/product/releases/health/#crash).
+
+- **Crash Free Session Rate**
+  Alerts when the number of crashed sessions exceeds the threshold you set. A session begins when a user starts the application and ends when it’s closed or sent to the background. A crash is when a session ends due to an error.
 
-- Crash Free Session Rate
-- Crash Free User Rate
+- **Crash Free User Rate**
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:76]:
```suggestion
- **Crash Free User Rate:**
```

File: src/docs/product/alerts/alert-types.mdx [hunk 32]
```diff
@@ -58,26 +58,61 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+- **Failure Rate**
+  Alerts on failure rate (the percentage of unsuccessful transactions). Sentry treats transactions with a status other than “ok,” “canceled,” and “unknown” as failures.
+
+- **Largest Contentful Display**
+  Alerts when the Largest Contentful Paint (LCP), which measures loading performance, is loading slower than expected. LCP marks the point when the largest image or text block is visible within the viewport.
+
+**TIP:** A fast LCP helps reassure the user that the page is useful. We recommend an LCP of less than 2.5 seconds.
+
+- **First Input Delay**
+  Alerts when First Input Delay (FID), which measures the response time when a user tries to interact with the viewport, is longer than expected.
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:103]:
```suggestion

- **Throughput:**
  Alerts when throughput (the total number of transactions in a project) reaches a threshold set by you within a specified period of time.

- **Transaction Duration:**
  Alerts when transaction durations meet certain conditions.

**TIP:** The conditions can be customized with flexible aggregates like percentiles, averages, and min/max.

- **Apdex:**
  Alerts on the Apdex score (the ratio of satisfactory, tolerable, and frustrated requests in a specific transaction or endpoint). Apdex is a metric used to track and measure user satisfaction based on your application response times.

- **Failure Rate:**
  Alerts on failure rate (the percentage of unsuccessful transactions). Sentry treats transactions with a status other than “ok,” “canceled,” and “unknown” as failures.

- **Largest Contentful Display:**
  Alerts when the Largest Contentful Paint (LCP), which measures loading performance, is loading slower than expected. LCP marks the point when the largest image or text block is visible within the viewport.

**TIP:** A fast LCP helps reassure the user that the page is useful. We recommend an LCP of less than 2.5 seconds.

- **First Input Delay:**
  Alerts when First Input Delay (FID), which measures the response time when a user tries to interact with the viewport, is longer than expected.
```

File: src/docs/product/alerts/alert-types.mdx (hunk 26 above)
Comment (shanamatthews) [reply src/docs/product/alerts/alert-types.mdx]: This is a good point - we could add anchors to these bullets so we can link to them directly.

Comment (lizokm) [reply src/docs/product/alerts/alert-types.mdx:73]: Hi there, great question. I think we should link the H3 Metric Alert type headers, so Errors, Sessions, and Performance. What do you think?

Comment (isabellaenriquez) [reply src/docs/product/alerts/alert-types.mdx]: I think that'd be the next best solution if we can't add anchors to the bullets!

File: src/docs/product/alerts/alert-types.mdx [hunk 33]
```diff
@@ -58,26 +58,59 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+
+- **Throughput**
+  Alerts when throughput (the total number of transactions in a project) reaches a threshold set by you within a specified period of time.
+
+- **Transaction Duration**
+  Alerts when transaction durations meet certain conditions.
+
+**TIP:** The conditions can be customized with flexible aggregates like percentiles, averages, and min/max.
+
+- **Apdex**
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:91]:
```suggestion
- **Throughput:**
  Alerts when throughput (the total number of transactions in a project) reaches a threshold set by you within a specified period of time.

- **Transaction Duration:**
  Alerts when transaction durations meet certain conditions.

**TIP:** The conditions can be customized with flexible aggregates like percentiles, averages, and min/max.

- **Apdex:**
```

File: src/docs/product/alerts/alert-types.mdx [hunk 34]
```diff
@@ -58,26 +58,59 @@ When you create an alert, all the displayed alert types (except “Issues”) ma
...
+
+- **Failure Rate**
+  Alerts on failure rate (the percentage of unsuccessful transactions). Sentry treats transactions with a status other than “ok,” “canceled,” and “unknown” as failures.
+
+- **Largest Contentful Display**
+  Alerts when the Largest Contentful Paint (LCP), which measures loading performance, is loading slower than expected. LCP marks the point when the largest image or text block is visible within the viewport.
+
+**TIP:** A fast LCP helps reassure the user that the page is useful. We recommend an LCP of less than 2.5 seconds.
+
+- **First Input Delay**
```
Comment (lizokm) [inline src/docs/product/alerts/alert-types.mdx:102]:
```suggestion
- **Failure Rate:**
  Alerts on failure rate (the percentage of unsuccessful transactions). Sentry treats transactions with a status other than “ok,” “canceled,” and “unknown” as failures.

- **Largest Contentful Display:**
  Alerts when the Largest Contentful Paint (LCP), which measures loading performance, is loading slower than expected. LCP marks the point when the largest image or text block is visible within the viewport.

**TIP:** A fast LCP helps reassure the user that the page is useful. We recommend an LCP of less than 2.5 seconds.

- **First Input Delay:**
```

**Review by isabellaenriquez — APPROVED** [review]
thanks so much!
